tests/backend/topology/test_random.py

```diff
@@ -10,9 +10,9 @@
 
```

Comment:
It'd be better to just parameterize the `static` variable than making it a fixture :+1: